helpers/inputTypeValidator.js

```diff
@@ -0,0 +1,9 @@
+export default (typeChecker = '', inputArray = '', next) => {
```

Comment:
There's something not tested in here.

Also, this looks to me like a middleware, hence, it should be in the middlewares directory, and it probably needs some refactoring. Either that or small little chunks of input validators can be created to account for input types of numbers, strings and arrays all individually.

Reply:
Okay, I'll check and refactor. Thanks